morf-oracle/src/main/java/org/alfasoftware/morf/jdbc/oracle/OracleMetaDataProvider.java

```diff
@@ -740,6 +744,32 @@ private String determineDefaultValue(String columnName) {
     return "";
   }
 
+
+  private String trimActualDefaultValue(final String actualDefaultValue) {

```

Comment:
It's fine, but this method does a bit more than just trimming. `getActualDefaultValue` or maybe 'parse'?

Reply:
It started as trimming, but then it kinda of exploded into some fancy stuff.. 👍 
I'll rename it..